pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
   'nestd == 0.3.1',
   'inquirerpy == 0.3.4',
 # conditional
-  "uvloop>=0.17.0; sys_platform == 'darwin' and platform_machine == 'aarch_64' and platform_machine == 'x86_64' and platform_machine == 'i686'"
+  "uvloop~=0.17.0; sys_platform != 'win32' and platform_python_implementation == 'CPython'"
```

Comment:
Hey @sansyrox ! Thanks for pushing this. Can you please explain the condition:

```
platform_python_implementation == 'CPython'
```

Thanks.

Reply:
Hey @banipreetr 👋 

This is needed when the users try to use robyn with `PyPy`. While we don't explicitly show support for PyPy on our repo. I am 70% sure that robyn works with PyPy lol